apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -558,6 +557,8 @@ func (m *ModelManagerAPI) CreateModel(ctx context.Context, args params.ModelCrea
 	// mode and don't make the calls so test can keep passing.
 	// THIS IS VERY TEMPORARY.
 	if m.modelService != nil {
+		args.CloudRegion = cloudRegionName
```

Comment:
@tlm We need to write the read-only model correctly now so I can get rid of this line.